### ISSUE TITLE
i build onnxruntime with c++ api, and create dynamic dll  and can c sharp call the dll directly?

### ISSUE THREAD

**Issue by cqray1990** [microsoft/onnxruntime#11544]
**Describe the bug**
A clear and concise description of what the bug is. To avoid repetition please make sure this is not one of the known issues mentioned on the respective release page.

**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- ONNX Runtime installed from (source or binary): 
- ONNX Runtime version:1.6
- Python version:3.6
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:10.2/8.0
- GPU model and memory:

i build onnxruntime with c++ api, and create dynamic dll  and can c sharp call the lib directly?, and my c++ code call onnxruntime dll, and then i build my code link onnxruntime dll to create an dynamic-dll, then i call the dll by c charp

**Comment by RyanUnderhill** [comment]
Duplicate of #11551 ?
